### ISSUE TITLE
Settings for the default pin location of opened images

### ISSUE THREAD

**Issue by sylph520** [sissilab/obsidian-image-toolkit#109]
When the figure is opened and pinned, it seems be pinned in the center of Obsidian. I'm wondering would there be options to configure this behavior (I did not find the settings). e.g., pinning to the top of Obsidian?  

Thanks.

**Comment by sissilab** [comment]
Yeah, the clicked image is currently displayed at the centre. Could I ask whether this placement would bother you, and if so, why?
